examples/pytorch/mnist-ddp/mnist.ipynb

```diff
@@ -0,0 +1,627 @@
+      "Train Epoch: 1 [25000/60000 (42%)]\tLoss: 0.485761\n",
+      "Train Epoch: 1 [26000/60000 (43%)]\tLoss: 0.641456\n",
+      "Train Epoch: 1 [27000/60000 (45%)]\tLoss: 0.389193\n",
+      "Train Epoch: 1 [28000/60000 (47%)]\tLoss: 0.374947\n",
+      "Train Epoch: 1 [29000/60000 (48%)]\tLoss: 0.366143\n",
+      "Train Epoch: 1 [30000/60000 (50%)]\tLoss: 0.393355\n",
+      "Train Epoch: 1 [31000/60000 (52%)]\tLoss: 0.425496\n",
+      "Train Epoch: 1 [32000/60000 (53%)]\tLoss: 0.377846\n",
+      "Train Epoch: 1 [33000/60000 (55%)]\tLoss: 0.317635\n",
+      "Train Epoch: 1 [34000/60000 (57%)]\tLoss: 0.527301\n",
+      "Train Epoch: 1 [35000/60000 (58%)]\tLoss: 0.592480\n",
+      "Train Epoch: 1 [36000/60000 (60%)]\tLoss: 0.372165\n",
+      "Train Epoch: 1 [37000/60000 (62%)]\tLoss: 0.404120\n",
+      "Train Epoch: 1 [38000/60000 (63%)]\tLoss: 0.468133\n",
+      "Train Epoch: 1 [39000/60000 (65%)]\tLoss: 0.522341\n",
+      "Train Epoch: 1 [40000/60000 (67%)]\tLoss: 0.402095\n",
+      "Train Epoch: 1 [41000/60000 (68%)]\tLoss: 0.514725\n",
+      "Train Epoch: 1 [42000/60000 (70%)]\tLoss: 0.437190\n",
+      "Train Epoch: 1 [43000/60000 (72%)]\tLoss: 0.373140\n",
+      "Train Epoch: 1 [44000/60000 (73%)]\tLoss: 0.339573\n",
+      "Train Epoch: 1 [45000/60000 (75%)]\tLoss: 0.256387\n",
+      "Train Epoch: 1 [46000/60000 (77%)]\tLoss: 0.459226\n",
+      "Train Epoch: 1 [47000/60000 (78%)]\tLoss: 0.408054\n",
+      "Train Epoch: 1 [48000/60000 (80%)]\tLoss: 0.460252\n",
+      "Train Epoch: 1 [49000/60000 (82%)]\tLoss: 0.373420\n",
+      "Train Epoch: 1 [50000/60000 (83%)]\tLoss: 0.295203\n",
+      "Train Epoch: 1 [51000/60000 (85%)]\tLoss: 0.474386\n",
+      "Train Epoch: 1 [52000/60000 (87%)]\tLoss: 0.411164\n",
+      "Train Epoch: 1 [53000/60000 (88%)]\tLoss: 0.359452\n",
+      "Train Epoch: 1 [54000/60000 (90%)]\tLoss: 0.309372\n",
+      "Train Epoch: 1 [55000/60000 (92%)]\tLoss: 0.356350\n",
+      "Train Epoch: 1 [56000/60000 (93%)]\tLoss: 0.363343\n",
+      "Train Epoch: 1 [57000/60000 (95%)]\tLoss: 0.482411\n",
+      "Train Epoch: 1 [58000/60000 (97%)]\tLoss: 0.353806\n",
+      "Train Epoch: 1 [59000/60000 (98%)]\tLoss: 0.274754\n",
+      "Training is finished\n"
+     ]
+    }
+   ],
+   "source": [
+    "import os\n",
+    "\n",
+    "# Set the Torch Distributed env variables so the training function can be run in the notebook\n",
+    "# See https://pytorch.org/docs/stable/elastic/run.html#environment-variables\n",
+    "os.environ[\"RANK\"] = \"0\"\n",
+    "os.environ[\"LOCAL_RANK\"] = \"0\"\n",
+    "os.environ[\"WORLD_SIZE\"] = \"1\"\n",
+    "os.environ[\"MASTER_ADDR\"] = \"localhost\"\n",
+    "os.environ[\"MASTER_PORT\"] = \"1234\"\n",
+    "\n",
+    "# Run the training function locally\n",
+    "train_fashion_mnist({\"epochs\": 1})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create the Kubeflow Training Client"
```

Comment:
Maybe we should also combine this section with the next cell, since users can do:
```py
from kubeflow.training import Trainer, TrainingClient

TrainingClient().train(...)

Reply:
Since we reuse the client to showcase how to list runtimes, watch logs, etc, that's probably better not to inline it here.